package.json

```diff
@@ -8,7 +8,7 @@
     "build": "sh -ac '. ./.env.${REACT_APP_ENV}; react-scripts build'",
     "build:test": "REACT_APP_ENV=test yarn build",
     "build:production": "REACT_APP_ENV=production yarn build",
-    "test": "react-scripts test --watchAll=false --detectOpenHandles --forceExit",
+    "test": "firebase emulators:exec --project aiscope-labelling-app-test --import mockData 'react-scripts test --watchAll=false --detectOpenHandles --forceExit'",
```

Comment:
\* Sorry for adding this comment after the PR is merged.

This change introduces a new dependency: `firebase-tools`, right?
With that, it is not possible to run the tests locally just by following the README instructions anymore.

![image](https://user-images.githubusercontent.com/15014647/178997705-1e11a0ec-cbb0-46db-961a-fa1f65066285.png)

I would say to either:
- include a instruction in the README file to add the new dependency as an environment dependency (globally)
```
yarn global add firebase-tools 
```
or
- add the dependency to the project, updating the `package.json` (I think this is the best option)
```
yarn add -D firebase-tools 
```

Please let me know if I am missing something, maybe you have already discussed about this.

@sendami, @analopezTW

Reply:
@marcosjordao , thanks for the feedback! 

You are totally right. We installed the dependency globally in order to initialize the emulator and make some tests with it. But maybe once the configuration of the emulator is in the repository, we don't need the global dependency anymore. I'll do some tests in order to check if adding the dependency to the package.json is enough and update the file or the readme depending on the results :)